src/DataCollector/SolrDataCollector.php

```diff
@@ -103,7 +113,9 @@ public function preExecuteRequest(SolariumPreExecuteRequestEvent $event): void
         $this->currentRequest = $event->getRequest();
         $this->currentEndpoint = $event->getEndpoint();
 
-        $this->logger->debug($event->getEndpoint()->getBaseUri().$this->currentRequest->getUri());
+        if ($this->logger) {
```

Comment:
Wonder if you can default to `NullLogger` to avoid this.

Reply:
Sure. Fixed.